pkg/ops/disputeOps.go

```diff
@@ -171,7 +172,7 @@ func getNonceSubmissions(ctx context.Context, valueBlock *big.Int, dispute *tell
 	return timedValues, nil
 }
 
-func List(ctx context.Context) error {
+func List(ctx context.Context, logger log.Logger) error {
```

Comment:
```suggestion
func List(ctx context.Context) error {
```

as discussed it looks like we don't need a logger here

Reply:
The issue here is that the function `simpleCmd` in main expect as a parameter a `func(context.Context, log.Logger) error`, if I change this I'll have to add another function in main to support this. I personally think leaving this here is less impactful than adding another function there. What do you think?